Mathlib/CategoryTheory/Skeletal.lean

```diff
@@ -92,9 +95,25 @@ instance : (fromSkeleton C).EssSurj where mem_essImage X := ⟨Quotient.mk' X, Q
 -- Porting note: named this instance
 noncomputable instance fromSkeleton.isEquivalence : (fromSkeleton C).IsEquivalence where
 
+theorem fromSkeleton_asEquivalence_inverse_obj (X : C) :
+    (fromSkeleton C).asEquivalence.inverse.obj X = ⟦X⟧ :=
+  Quotient.eq_mk_iff_out.mpr ⟨(fromSkeleton C).asEquivalence.counitIso.app X⟩
+
 /-- The equivalence between the skeleton and the category itself. -/
 noncomputable def skeletonEquivalence : Skeleton C ≌ C :=
-  (fromSkeleton C).asEquivalence
+  (fromSkeleton C).asEquivalence.changeInverse <|
+    Functor.isoCopyObj _ _ (eqToIso <| fromSkeleton_asEquivalence_inverse_obj C ·)
```

Comment:
Hmm, looking at this now it seems much more sensible to define both directions of this equivalence explicitly... What do you think? I don't mind if it's a todo in this file instead of in this PR